pkg/proto/ir.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
```

Comment:
If the codegen is going to template-fill the IR into generated code, why do we need to redefine the IR in protobuf?